src/LayoutManager/LayoutManager.spec.ts

```diff
@@ -772,6 +773,19 @@ describe('Layout Manager', () => {
       expect(child.getCenterPoint()).toMatchObject(group.getCenterPoint());
     });
 
+    it('should subscribe objects on initialization', () => {
+      const child = new FabricObject({
+        width: 200,
+        height: 200,
+        strokeWidth: 0,
+      });
+      jest.spyOn(child, 'toJSON').mockReturnValue('child');
+      const group = new Group([child]);
+      expect(
+        Array.from(group.layoutManager['_subscriptions'].keys())
+      ).toMatchObject([child]);
+    });
```

Comment:
i do not understand this test and why has to involve toJSON of fabricJS objects.

Reply:
I mean i understand you want  to test the _subscription part but why do you have to mock toJSON. When we run code toJSON is not mockable.

Reply:
That is exactly why I mock it, to avoid huge useless dumps
Make the test fail and see how elegant the dump is. It uses the mocked value in the error message.
It should be a test util we use to register all fabric instances in a test. I used it in one of my CI pending PRs but did not expose it as a util.
```
registerTestItem({ a, b, c, d})
```
Then the var name is used in test output

Reply:
So to explain.
The test was failing and the dump was unreadable so I added it for the future in case it does